modules/engage-paella-player-7/src/plugins/org.opencast.paella.multiVideoDynamicLayout.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * Licensed to The Apereo Foundation under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ *
+ * The Apereo Foundation licenses this file to you under the Educational
+ * Community License, Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License. You may obtain a copy of the License
+ * at:
+ *
+ *   http://opensource.org/licenses/ecl2.txt
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ *
+ */
+
+import {VideoLayout} from 'paella-core';
+
+export default class MultiVideoDynamicLayout extends VideoLayout {
+  get identifier() {
+    return 'multiple-video-dynamic';
+  }
+
+  get layoutType() {
+    return 'dynamic';
+  }
+
+  async load() {
+    this.player.log.debug('Multi video layout loaded');
+  }
+
+  getValidStreams(streamData) {
+    // Ignore content of streamData
+    return [streamData];
+  }
+
+  // eslint-disable-next-line no-unused-vars
+  getValidContentIds(streamData) {
+    // Ignore content of streamData
+    return this.validContentIds;
+  }
+
+  // eslint-disable-next-line no-unused-vars
+  getLayoutStructure(streamData, contentId) {
```

Comment:
Can't you just not define unnecessary parameters? I remember that you can have more parameters in a JS function than are defined. So, this should work:

```suggestion
  getLayoutStructure(streamData) {
```